assets/quick-add.css

```diff
@@ -0,0 +1,200 @@
+  display: flex;
+  overflow-y: auto;
+  padding: 2.5rem;
+  height: 100%;
+}
+
+.quick-add-modal__content-info > * {
+  height: auto;
+  margin: 0 auto;
+  max-width: 100%;
+  width: 100%;
+}
+
+@media screen and (max-width: 749px) {
+  .quick-add-modal__content {
+    bottom: 3.2rem;
+  }
+  
+  .quick-add-modal__content-info > * {
+    max-height: 100%;
+  }
+}
+
+.quick-add-modal__toggle {
+  background-color: rgb(var(--color-background));
+  border: 0.1rem solid rgba(var(--color-foreground), 0.1);
+  border-radius: 50%;
+  color: rgba(var(--color-foreground), 0.55);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  padding: 1.2rem;
+  z-index: 5;
+  width: 4rem;
+  position: fixed;
+  top: 1.5rem;
+  right: 2rem;
+}
+
+.quick-add-modal__toggle:hover {
+  color: rgba(var(--color-foreground), 0.75);
+}
+
+.quick-add-modal__toggle .icon {
+  height: auto;
+  margin: 0;
+  width: 2.2rem;
+}
+
+.quick-add-modal__content {
+  max-width: var(--page-width);
+}
+
+quick-add-modal .product:not(.featured-product) .product__view-details {
+  display: block;
+}
+
+.quick-add-hidden,
+quick-add-modal .product__modal-opener:not(.product__modal-opener--image),
+quick-add-modal .product__media-item:not(:first-child) {
+  display: none !important;
+}
+
+quick-add-modal .product__media-list .deferred-media {
+  display: block;
+  width: 100%;
+}
+
+@media screen and (max-width: 749px) {
+  quick-add-modal .product__media-wrapper slider-component:not(.thumbnail-slider--no-slide) {
+      margin-left: -2.5rem;
+      margin-right: -2.5rem;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  quick-add-modal .product--small:not(.product--no-media) .product__media-wrapper,
+  quick-add-modal .product--medium:not(.product--no-media) .product__media-wrapper,
+  quick-add-modal .product--large:not(.product--no-media) .product__media-wrapper {
+    max-width: 45%;
+    width: calc(45% - var(--grid-desktop-horizontal-spacing) / 2);
+  }
+
+  quick-add-modal .product--small:not(.product--no-media) .product__info-wrapper,
+  quick-add-modal .product--medium:not(.product--no-media) .product__info-wrapper,
+  quick-add-modal .product--large:not(.product--no-media) .product__info-wrapper {
+    padding-left: 4rem;
+    max-width: 54%;
+    width: calc(54% - var(--grid-desktop-horizontal-spacing) / 2)
+  }
+
+  quick-add-modal .thumbnail-slider .thumbnail-list.slider--tablet-up {
+    display: none;
+  }
+}
+
+quick-add-modal .page-width {
+  padding: 0;
+}
+
+quick-add-modal .product__title > h1 {
+  display: none;
+}
+
+quick-add-modal .product__title > a {
+  display: block;
+  text-decoration: none;
+}
+
+quick-add-modal .product__title > a:hover {
+  color: rgb(var(--color-foreground));
+  text-decoration: underline;
+  text-underline-offset: 0.2rem;
+  text-decoration-thickness: 0.3rem;
+}
+
+quick-add-modal .product__view-details {
+  margin-bottom: 4rem;
+}
+
+.quick-add__submit {
+  padding: 0 2rem;
```

Comment:
I wonder if we should have some padding top here. When I grabbed the translation from another language it looked like this: 
![](https://screenshot.click/21-18-hpbbj-2v1sm.png)